src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1633,17 +1661,166 @@ public void checkIfStatusDateChangesWhenItemStatusUpdated()
+      .put("status", new JsonObject().put("name", Status.Name.CHECKED_OUT.value()));
+
+    itemsClient.replace(id, itemWithUpdatedStatus);
+
+    Response updatedItemResponse = itemsClient.getById(id);
+    JsonObject updatedStatus = updatedItemResponse.getJson().getJsonObject("status");
+
+    assertThat(updatedStatus.getString("name"), is(Status.Name.CHECKED_OUT.value()));
+    assertThat(updatedStatus.getString("date"), withinSecondsBeforeNowAsString(seconds(2)));
+
+    JsonObject itemWithUpdatedCallNumber = updatedItemResponse.getJson().copy()
+      .put("itemLevelCallNumber", "newItemLevelCallNumber");
+
+    itemsClient.replace(id, itemWithUpdatedCallNumber);
+    JsonObject itemWithUpdatedCallNumberResponse = itemsClient.getById(id).getJson();
+
+    assertThat(itemWithUpdatedCallNumberResponse.getString("itemLevelCallNumber"),
+      is("newItemLevelCallNumber"));
+
+    String oldStatusDate = updatedStatus.getString("date");
+    String newStatusDate = itemWithUpdatedCallNumberResponse.getJsonObject("status").getString("date");
+
+    assertThat(oldStatusDate, is(newStatusDate));
+  }
+
+  @Test
+  public void statusUpdatedDateIsNullOnSubsequentUpdates() throws Exception {
+    UUID holdingsRecordId = createInstanceAndHolding(mainLibraryLocationId);
+    UUID id = UUID.randomUUID();
+    JsonObject itemToCreate = smallAngryPlanet(id, holdingsRecordId)
+      .put("status", new JsonObject().put("name", "Available"));
+    createItem(itemToCreate);
+
+    JsonObject createdItem = getById(id).getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"), is("Available"));
+    assertThat(createdItem.getJsonObject("status").getString("date"), nullValue());
+
+    JsonObject firstUpdateItem = createdItem.copy()
+      .put("itemLevelCallNumber", "newItCn");
+
+    itemsClient.replace(id, firstUpdateItem);
+
+    JsonObject firstUpdatedItemResponse = itemsClient.getById(id).getJson();
+
+    assertThat(firstUpdatedItemResponse.getString("itemLevelCallNumber"),
+      is("newItCn"));
+    assertThat(firstUpdatedItemResponse.getJsonObject("status").getString("name"),
+      is("Available"));
+    assertThat(firstUpdatedItemResponse.getJsonObject("status").getString("date"),
+      nullValue());
+
+    JsonObject secondUpdateItem = firstUpdatedItemResponse.copy()
+      .put("temporaryLocationId", onlineLocationId.toString());
+
+    itemsClient.replace(id, secondUpdateItem);
+
+    JsonObject secondUpdatedItemResponse = itemsClient.getById(id).getJson();
+
+    assertThat(secondUpdatedItemResponse.getString("temporaryLocationId"),
+      is(onlineLocationId.toString()));
+    assertThat(secondUpdatedItemResponse.getJsonObject("status").getString("name"),
+      is("Available"));
+    assertThat(secondUpdatedItemResponse.getJsonObject("status").getString("date"),
+      nullValue());
+  }
+
+  @Test
+  public void canUpdateItemWithCopyNumber() throws Exception {
```

Comment:
Does this need to be a specific test, or can it be included in the general updating of an item?

Reply:
Seems we don't have a separate test case for updating an item, only for creating.

Reply:
Hmm, ok, I wonder if this test could be changed to a more general can replace item with new properties test?